PacketDelivery-VC-ReferenceExample/data-consumer/README.md

```diff
@@ -0,0 +1,173 @@
+# Data Service Consumer with verifiable credentials
+
+This describes how to setup the environment of a data service consuming organisation within the i4Trust trusted data space with the example of the fictitious organisations
+
+* Packet Delivery Company being the service provider
+* Happy Pets Inc. being the consuming organisation
+
+Needed for issuing credentials that can be used against the authentication of the service provider's environment are the following
+* [WaltId](https://github.com/walt-id/waltid-ssikit)
+* [Keycloak](https://www.keycloak.org/) with the [VC Issuer plugin](https://github.com/wistefan/keycloak-vc-issuer)
+
+Furthermore are needed
+* VC compatible Wallet for executing the OpenIDConnect flow
+* iSHARE Satellite as trust anchor for sharing trusted parties
+
+![Components](./img/components.png "Components")
+
+Note that there is not an example of a Shop System provided. In the use case example, it is assumed that there are already 
+users registered at Happy Pets and which have placed orders. More precisely, this means that there are already 
+existing delivery order entities within the Context Broker of 
+the [Data Service Provider](../Data-Service-Provider).
+
+Furthermore it is required that there is access to an iSHARE Satellite instance and an iSHARE-compliant Authorisation 
+Registry which stores the user's access policies.
+
+In the following it is assumed that the components will be externally available via the domain `domain.org`. 
+
+## Installation
+
+It is recommended to at least change the "route.host" property of the keycloak chart to adjust to the own test environment.
+
+To install the components execute the following commands
+
+```
+helm dependency build waltid/
+helm install -ndemo waltid ./waltid/ -f ./waltid/values.yaml
+helm dependency build keycloak/
+helm install -ndemo keycloak ./keycloak/ -f ./keycloak/values.yaml
+```
+
+## Overview of the users/credentials
+
+With the predefined value files, users and DIDs for the necessary parties are imported as follows:
+
+|   |   |
+|---|---|
```

Comment:
Did it generate so similar did's or is one of them just made up?

Reply:
Just made that one up but will probably adjust it in the producer tutorial PR